packages/lesswrong/lib/collections/revisions/schema.ts

```diff
@@ -102,8 +104,7 @@ const schema: SchemaType<DbRevision> = {
   },
   originalContents: {
     type: ContentType,
-    viewableBy: ['guests'],
-    editableBy: ['members']
```

Comment:
@jimrandomh, was the members permission on originalContents serving a purpose? It seemed superficially like a security hole, and my impression is that originalContents is updated via callback, and ckEditor autosaves (which is a rawUpdate). It seemed unlikely we want the permissions here to be "all members" but I'm not sure why it was set up this way in the first place.

Reply:
The collection does not have mutation resolvers, so it can't be edited via the CRUD API (even by admins), so this permissions line has no server-side/security implications. But vulcan-forms introspects on permissions, so it's possible this matters for that reason. But if you tested end-to-end creating and editing something with an editable, it should be fine.